dGame/dGameMessages/GameMessages.cpp

```diff
@@ -3861,7 +3862,7 @@ void GameMessages::SendDisplayChatBubble(LWOOBJID objectId, const std::u16string
 void GameMessages::SendSetMountInventoryID(Entity* entity, const LWOOBJID& objectID, const SystemAddress& sysAddr) {
 	CBITSTREAM;
 	CMSGHEADER;
-
+	Game::logger->Log("GameMessages", "set moint inventory ID!");
```

Comment:
I am going to stop leaving these comments now but like, there are too many logs, helpful and not.

Reply:
This is a draft pr.  Many of these are just for testing so I expect these will be cleaned up when this is merged

Reply:
yes